thoth/package_extract/image.py

```diff
@@ -546,6 +546,47 @@ def get_image_size(path: str) -> int:
     return total_size
 
 
+def _get_cuda_version(path: str) -> dict:
+    """Get the cuda version."""
+    res = dict()
+    # gathering version from version.txt file
+    version_path = path + "/usr/local/cuda/version.txt"
+    if os.path.isfile(version_path):
+        with open(version_path, "r") as f:
+            output = f.read()
+            lines = output.split()
+            if len(lines) != 3:
```

Comment:
Here, we are explicitly checking whether `length` is equal to 3, otherwise we would skip it.

Reply:
See my comment above, this solution is more prone to issues.